operations/security_and_authentication/Troubleshoot_Kyverno_Configuration_manually.md

```diff
@@ -0,0 +1,25 @@
+# How to troubleshoot Kyverno configuration manually
+
+## Check if Kyverno pods are up and running
+
+Run the script with -p option.
+
+```bash
+/opt/cray/tests/install/livecd/scripts/k8s_kyverno_pods_running.sh -p
+```
+
+This will list the expected Kyverno pods in running state.
```

Comment:
Running this command on a system with 1.3.0-beta.74 does not show Kyverno pods in a running state, it only prints `PASS`. This irrespective of whether or not the `-p` is provided. The documentation should be clear on expected output (e.g., the only time `PASS` isn't printed is when an error exists).

Reply:
Stale comment, marking as closed.